orbit/tools/build/build-windows.go

```diff
@@ -0,0 +1,265 @@
+	} else {
+
+		defer os.Remove(outPath)
+		// now we can build the binary
+		if err := buildTargetBinary(*flagCmdDir, *flagVersion, *flagOutputBinary); err != nil {
+			zlog.Fatal().Err(err).Msg("error building binary")
+			os.Exit(1)
+		}
+
+		zlog.Info().Msgf("Successfully built %s", *flagOutputBinary)
+	}
+}
+
+// createVersionInfo returns a VersionInfo struct pointer to be used to generate the 'resource.syso'
+// metadata file (see writeResourceSyso).
+func createVersionInfo(vParts []string, iconPath string, manifestPath string) (*goversioninfo.VersionInfo, error) {
+	vIntParts := make([]int, 0, len(vParts))
+	for _, p := range vParts {
+		v, err := strconv.Atoi(p)
+		if err != nil {
+			return nil, fmt.Errorf("error parsing version part %s: %w", p, err)
+		}
+		vIntParts = append(vIntParts, v)
+	}
+	version := strings.Join(vParts, ".")
+	copyright := fmt.Sprintf("%d Fleet Device Management Inc.", time.Now().Year())
+
+	// Taken from https://github.com/josephspurrier/goversioninfo/blob/master/testdata/resource/versioninfo.json
+	langID, err := strconv.ParseUint("0409", 16, 16)
+	if err != nil {
+		return nil, errors.New("invalid LangID")
+	}
+	// Taken from https://github.com/josephspurrier/goversioninfo/blob/master/testdata/resource/versioninfo.json
+	charsetID, err := strconv.ParseUint("04B0", 16, 16)
+	if err != nil {
+		return nil, errors.New("invalid charsetID")
+	}
+
+	result := goversioninfo.VersionInfo{
+		FixedFileInfo: goversioninfo.FixedFileInfo{
+			FileVersion: goversioninfo.FileVersion{
+				Major: vIntParts[0],
+				Minor: vIntParts[1],
+				Patch: vIntParts[2],
+				Build: vIntParts[3],
+			},
+			ProductVersion: goversioninfo.FileVersion{
+				Major: vIntParts[0],
+				Minor: vIntParts[1],
+				Patch: vIntParts[2],
+				Build: vIntParts[3],
+			},
+			FileFlagsMask: "3f",
+			FileFlags:     "00",
+			FileOS:        "040004",
+			FileType:      "01",
+			FileSubType:   "00",
+		},
+		StringFileInfo: goversioninfo.StringFileInfo{
+			Comments:         "Fleet osquery",
+			CompanyName:      "Fleet Device Management (fleetdm.com)",
+			FileDescription:  "Orbit osquery runtime and autoupdater",
+			FileVersion:      version,
+			InternalName:     "",
+			LegalCopyright:   copyright,
+			LegalTrademarks:  "",
+			OriginalFilename: "",
+			PrivateBuild:     "",
+			ProductName:      "Fleet osquery",
+			ProductVersion:   version,
+			SpecialBuild:     "",
+		},
+		VarFileInfo: goversioninfo.VarFileInfo{
+			Translation: goversioninfo.Translation{
+				LangID:    goversioninfo.LangID(langID),
+				CharsetID: goversioninfo.CharsetID(charsetID),
+			},
+		},
+		IconPath:     iconPath,
+		ManifestPath: manifestPath,
+	}
+
+	return &result, nil
+}
+
+// sanitizeVersion returns the version parts (Major, Minor, Patch and Build), filling the Build part
+// with '0' if missing. Will error out if the version string is missing the Major, Minor or
+// Patch part(s).
+func sanitizeVersion(version string) ([]string, error) {
```

Comment:
Duplicate of https://github.com/fleetdm/fleet/pull/10783/files#diff-95043ac955352f5e6fe86545b3a5cebae5caa643a9e91da888d302e6d1c7fb98R336 ?

Reply:
although they have the same name, there are some minor differences between the functions, such as the file description information and the ability to embed an icon as part of the versioninfo resource (new parameter on the function signature). We might want to keep these two separate for the time being as there could be different customizations for the orbit files (such as locale support) and the installer file